Dockerfile

```diff
@@ -3,9 +3,10 @@ FROM registry.cn-hangzhou.aliyuncs.com/alinux/alinux3 as builder
 WORKDIR /usr/src/verdictd
 
 ENV RATS_TLS_COMMIT 5de6fc3
-ENV VERDICTD_COMMIT 1d632be
+ENV VERDICTD_COMMIT 822ed01
 
 COPY . .
+RUN git reset --hard ${VERDICTD_COMMIT}
```

Comment:
I tried to build this, but git doesn't seem to be installed:

```
 => ERROR [builder  4/12] RUN git reset --hard 822ed01                                                                                                                                                                                                 0.2s
------
 > [builder  4/12] RUN git reset --hard 822ed01:
#11 0.165 /bin/sh: git: command not found
```

You could install git within the `yum install` below and run the` git reset` afterwards.